test/test.js

```diff
@@ -133,4 +133,34 @@ describe('chai-shallow-deep-equal', function() {
           'Expected "42" to equal ' + '"2014-09-29T20:00:00.000Z" at path ".".'
       );
     });
+
+    it('success on missing properties', function() {
+        new chai.Assertion({a: 10}).to.be.shallowDeepEqual({a: 10, b: null});
```

Comment:
`null` and `undefined` are different concept, can you elaborate a workflow where this kind of test would make sense?